### DISCUSSION TITLE
Distorted sound since Beta 7 (still in Beta 8)

### DISCUSSION THREAD

**Discussion by Bholtland** [apohl79/audiogridder#944] [Help]
Hi,

Since Beta 7 the sound I get back from AudioGridder is very distorted. In Beta 6 this was not the case. It sounds like a buffer issue to me.
I'm using FL Studio 20.9.2 on Windows 11.

The only way I could fix it till now is by turning on "Use fixed size buffers" for AudioGridder plugin but FL Studio does not recommend turning that on. I also tried every setting in AudioGridder but that did not work.

Is there any way to get around this? I'm not certain if this is considered a bug, but I'm happy to provide more information.

Thanks in advance!

**Comment by apohl79** [comment]
Please try beta9.

**Reply by Bholtland** [reply]
Just tried it, but unfortunately it did not solve the issue. Thanks for your reply!

**Reply by apohl79** [reply]
Ok, is this happening with the FX or instrument plugin or both? Does it happen with no buffering in AG (0 blocks) and with buffering enabled?

**Reply by Bholtland** [reply]
There seems to be no difference between FX or instruments plugins. Both have the same issue.

But... now that I've tested it again with buffer size disabled, it works! I must have missed something this morning when testing all the buffer sizes. Thanks for your help and your awesome work.

**Reply by apohl79** [reply]
Ok, so it is an issue with the buffering. I will have a look and see if I can reproduce this with an FL trial or so. Thx!

**Reply by Bholtland** [reply]
Thanks, let me know if I can help with any testing or so.

**Reply by apohl79** [reply]
will be fixed in the next version, thx for reporting
